zookeeper-server/src/main/java/org/apache/zookeeper/server/auth/znode/groupacl/ZkClientUriDomainMappingHelper.java

```diff
@@ -125,8 +126,10 @@ private void parseZNodeMapping() {
         try {
           List<String> clientUris =
               zks.getZKDatabase().getChildren(rootPath + "/" + domainName, null, null);
-          clientUris.forEach(
-              clientUri -> newClientUriToDomainNames.computeIfAbsent(clientUri, k -> new HashSet<>()).add(domainName));
+          clientUris.forEach(clientUri -> {
+              LOG.debug("ZkClientUriDomainMappingHelper::parseZNodeMapping(): Adding client uri mapping:{} {}", clientUri, domainName);
```

Comment:
Let's make clear which is which?

`(..."domainName: {}, clientUri: {}", domainName, clientUri);`